docs/content/FAQ/_index.md

```diff
@@ -37,6 +38,14 @@ If you want to see what has changed between versions, refer to the [releases pag
 
 The Infection Monkey agent shuts off either when it can't find new victims or it has exceeded the quota of victims as defined in the configuration.
 
+## Is Infection Monkey a malware/virus?
+
+Infection Monkey is not malware, but it uses similar techniques to safely simulate malware on your network.
+
+Because of this, Monkey gets flagged as malware by some antivirus solutions during the installation. If this happens,
```

Comment:
```suggestion
Because of this, The Infection Monkey gets flagged as malware by some antivirus solutions during the installation. If this happens,
```